source_x.tsv

```diff
@@ -0,0 +1,1000 @@
+-1.5070	-1.2101	-1.8913	-1.2325	-1.7648	-1.9574	-1.4571	-1.5716	0.1343	0.1558	0.5140	-0.3694	0.1643	0.0137	-0.3895	0.2263	0.7118	0.6599	-0.0339	0.0403	0.5729	0.5825	-0.4915	0.2278	0.3535	0.5253	0.5542	0.4414	-0.0710	0.1367	0.3718	0.0110	0.4017	1.3144	1.2567	0.5810	1.1481	1.5687	0.8800	0.5468	0.8844	0.7410	0.8354	0.6151	0.7135	0.6974	0.9243	1.0096	0.7768	0.1828	0.1796	-0.0027	0.1659	0.5745	0.1225	0.1250	0.2296	0.0338	0.1482	0.0772	0.0403	0.0356	0.7710	0.2868	0.4628	0.0673	0.3767	0.5237	-0.4041	0.0639	0.3014	-0.1758	0.8723	0.8673	0.7620	0.6000	0.9484	0.4426	0.1033	1.1750	0.3598	0.6782	-1.5255	0.8949	0.8640	-1.5875	-0.2502	0.3891	0.8485	0.0829	-1.0225	-0.1950	0.1583	0.9355	-1.9144	0.0376	0.3483	-0.2501	-0.4496	0.5398
```

Comment:
Would you mind putting these data files in a subdirectory (like `./simulated_output/source_x.tsv`, and the same for the other data files you generated)? That just makes it a bit clearer that these are output from the data generation code, and not data files that are necessary for the code to run.

Reply:
Sure thing.